Guide Examples/Policy Activity (V3)_2.16.840.1.113883.10.20.22.4.61/Policy Activity (V3) Example.xml

```diff
@@ -1,11 +1,36 @@
 <act classCode="ACT" moodCode="EVN">
   <templateId root="2.16.840.1.113883.10.20.22.4.61" extension="2015-08-01" />
-  <id root="3e676a50-7aac-11db-9fe1-0800200c9a66" />
-  <code code="12" displayName="Medicare Secondary Working Aged Beneficiary or Spouse with Employer Group Health Plan" 
+  <!-- Group Identifier - root represents the payer's OID for their group numbers, i.e. Blue Cross, United, etc.; extension represents the actual group identifier -->
+  <!-- From a practical standpoint, the id records here are not "usable" for defining the id's of the act, discuss with Benjamin Flessner? -->
+  <id assigningAuthorityName="XYZ Insurance" root="1.1.1.3.4.5" extension="320221" /> <!-- This is the Group # -->
+  <id assigningAuthorityName="XYZ Insurance" root="2.9.3.4.192.1" extension="394952" /> <!-- This is the RxGRP -->
+  <id assigningAuthorityName="XYZ Insurance" root="2.9.3.4.195" extension="131" /> <!-- This is the Plan # -->
+  <sdtc:identifiedBy typeCode="REL">
```

Comment:
Okay, this is a fair bit more important than the typo: the current version of the schema (where we added sdtc:identifiedBy to a bunch of places) still doesn't allow it under the act element. I think maybe we limited where it was allowed to be because of the RIM? In any case, this current example fails schema validation. ):